### ISSUE TITLE
Server software uninstalls itself with no user interaction.

### ISSUE THREAD

**Issue by BuffTucker** [RainwayApp/bug-tracker#121]
I played with Rainway from work all day today, except for the last couple of hours of my shift because I expected to come home and update the server software tomorrow and give it another go tomorrow.

I got home and the server was no longer running.

I opened the start menu to trigger the manual update and the folder was missing.

I navigated to the installation directory and all files are gone except for a config file and a log file.  Aaaaand I just checked the log file because I thought posting it here might be helpful, but the log file is an empty text document.

Long story short- server worked fine all day, I arrived home and the server had uninstalled itself with absolutely no user interaction whatsoever.

**Comment by olivieralvespro** [comment]
yes very wierd happened to me too (but only once so far)

**Comment by IkeKap** [comment]
Has happened to me three times so far. If I remember correctly, the majority of the times it happened for me, I was using the taskbar update option versus the the start menu option

**Comment by Codeusa** [comment]
@IkeKap if that is the case, I fixed this in the latest update.
